pages/linux/zenity.md

```diff
@@ -0,0 +1,23 @@
+# zenity
+
+> Display dialogs from the command line/shell scripts, and return user-inserted values or 1 if error.
+
+- Display the default question dialog:
+
+`zenity --quesion`
+
+- Display an info dialog displaying the text "Hello!":
+
+`zenity --info --text="{{Hello!}}"`
+
+- Display a name/password form and output the data separated by ';':
+
+`zenity --forms --add-entry="{{Name}}" --add-password="{{Password}}" --separator="{{;}}"`
+
+- Display a file selection form in which the user can only select directories:
+
+`zenity --file-selection --directory`
+
+- Display a progress bar and update its text message and progress percent:
```

Comment:
How about - 
`Display a progress bar which updates its message every second and shows a progress percent.` 

Seems more related to the command below.